### DISCUSSION TITLE
Train NER on our specific data, requirements

### DISCUSSION THREAD

**Discussion by jurys22** [urchade/GLiNER#48] [Q&A]
I would like to use this model with my current dataset which is multilingual and strictly domain-specific. I would like to use it offline. Here are my questions regarding a re-training of the model:

- Is it possible to run it offline preserving privacy?
- How many examples should I label to obtain a good result?
- Do you have any other advice?

Thank you

**Comment by urchade** [comment]
> Is it possible to run it offline preserving privacy?

sure, you can

> How many examples should I label to obtain a good result?

It depends on many factors. Do you mean training from scratch or fine-tuning existing Gliner ?
For fine tuning, you can try even with hundreds of examples.
